include/UIKit/UIKitTypes.h

```diff
@@ -48,7 +48,7 @@ typedef NS_ENUM(NSUInteger, NSWritingDirection) {
 };
 
 enum {
-    UITextAlignmentLeft,
+    UITextAlignmentLeft = 0,
     UITextAlignmentCenter,
```

Comment:
This makes UI/NS unable to be coerced, which I don't believe is intended behavior #WontFix